downstream/modules/automation-hub/proc-downloading-signature-public-keys.adoc

```diff
@@ -10,7 +10,7 @@ After you sign and approve collections, download the signature public keys from
 
 . Log in to your private {HubName} instance in the {HubName} UI.
 
-. In the left navigation, click *Signature Keys*. The Signature Keys dashboard displays a list of multiple keys: collections and container images.
+. In the left navigation, click btn:[Signature Keys]. The Signature Keys dashboard displays a list of multiple keys: collections and container images.
```

Comment:
Using directional information such as left, right...is a violation. I also think this is a menu selection, suggest changing to:

In the navigation pane, select menu:Signature Keys[].